packages/backend/src/guards/firebase.auth.ts

```diff
@@ -28,7 +31,12 @@ export class FirebaseAuthStrategy extends PassportStrategy(
     if (!firebaseUser) {
       throw new UnauthorizedException();
     }
-
+    // https://stackoverflow.com/questions/18214635/what-is-returned-from-mongoose-query-that-finds-no-matches
+    if (userStoreService.findOneByFirebaseId(firebaseUser.uid) === null) {
```

Comment:
Should this be awaited?

```suggestion
    if (await userStoreService.findOneByFirebaseId(firebaseUser.uid) === null) {
```